halpybot/packages/edsm/edsm.py

```diff
@@ -383,6 +383,8 @@ def landmarks(self):
         if self._landmarks:
             return self._landmarks
         landmark_target = Path() / "data" / "edsm" / "landmarks.json"
+        if not landmark_target.is_file():
+            raise FileNotFoundError
```

Comment:
[semantics] technically all you can determine with this check is that it wasn't a  file, not that it doesn't exist.